tests/test_nodes.py

```diff
@@ -359,30 +359,41 @@ class DummyEnum(Enum):
 
 @pytest.mark.parametrize("is_optional", [True, False])  # type: ignore
 @pytest.mark.parametrize(  # type: ignore
-    "type_,value, expected_type",
+    "ref_type, type_, value, expected_type",
     [
-        (Any, 10, AnyNode),
-        (DummyEnum, DummyEnum.FOO, EnumNode),
-        (int, 42, IntegerNode),
-        (float, 3.1415, FloatNode),
-        (bool, True, BooleanNode),
-        (str, "foo", StringNode),
+        (Any, Any, 10, AnyNode),
+        (DummyEnum, DummyEnum, DummyEnum.FOO, EnumNode),
+        (int, int, 42, IntegerNode),
+        (float, float, 3.1415, FloatNode),
+        (bool, bool, True, BooleanNode),
+        (str, str, "foo", StringNode),
```

Comment:
Do you support an IntegerNode with Any refrtype?

Reply:
No.